second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+researchers to create new and significant results. And, the paper serves as a
+good model of how to make all code and data available, making the results easily
+reproducible and expandable to other researchers curious about the behaviour of
+additional strategies.
+\end{verbatim}
+    \item 
+
+\begin{verbatim}
+The reported experiments fill an important hole in the understanding of IPD
+evolution and are sorely needed. The authors also take pains to ensure the data
+are publicly available, which is highly appreciated.
+\end{verbatim}
+\end{itemize}
+
+Most requests were on the subject of
+presentation of the results as well as requests for specific pieces of further
+analysis. All of these were addressed.
+
+This latest set of comments from the reviewers are mainly of two types:
+
+\begin{enumerate}
+    \item Further minor requests for presentation modifications (specifically
+        the remaining reviewer from the first round only made requests of this
+        type).
+    \item Questions about the suitability/impact of the work.
+\end{enumerate}
+
+As we detail below we have addressed all requests of the first type. For the
+requests of the second type, from the two new reviewers who are perhaps not
+experts in the IPD, we have made some modifications to the manuscript but our
+main responses are to answer the questions posed by the reviewers.
+
+Specifically we describe how the Moran process is a well studied model of
+evolutionary processes and that this paper is a major contribution to the
+literature. It can be thought of as a tournament on Moran processes equivalent
+to the seminal work by Robert Axelrod in the 1980s.
+
+Robert Axelrod's work on the Prisoner's Dilemma tournaments (described in
+the literature review) which has been cited more than 34,000 times considered
+pair wise matches between strategies. Our work here provides a much more robust
+type of tournament where strategies are pitted against one another in an actual
+evolutionary process (as opposed to a single one of interaction).
+
+\section{Comments from first referee}
+
+\begin{verbatim}
+   Figures are much improved! It was a good idea to move some of them to the
+Supplementary Material. I still think the title is not representative of the
+work -- you should reconsider at least changing the subtitle. You are not
+studying the Moran process; you are studying Prisoner's Dilemma strategies.  I
+\end{verbatim}
+
+Thank you for agreeing to review the paper again.
+
+We have changed the subtitle to be: `Empirical Study of strategies in the Moran
+process for the Iterated Prisoner's
+
+\begin{verbatim}
+also still think the description on page 3 of how you found the TF1, TF2, and
+TF3 strategies is confusing. It has a frustrating level of detail. You should
+either have less detail or more detail. This is not a crucial point, however.
+\end{verbatim}
+
+The description of how we obtain the strategies is quite minimalistic.
```

Comment:
Let's also mention that the training code is available and documented for further use.